### PR TITLE
feat: use ws-tool support deflate

### PR DESCRIPTION
**Pull request by AdamEECS**

### CODE REVIEW

File: taos-ws/src/query/asyn.rs [hunk 1]
```diff
@@ -1122,193 +1257,396 @@ impl AsyncQueryable for WsTaos {
...
-        v: Option<bool>,
+        // loop n times to test persistent connection
+        // do not run in ci env
+        let n = 100;
+        let interval = Duration::from_secs(3);
+        for _ in 0..n {
+            assert_eq!(
+                client
+                    .exec(format!(
+                        r#"insert into {db}.tb1 using {db}.stb1 tags('{{"key":"数据"}}')
```
Comment (sheyanjie-taos) [inline taos-ws/src/query/asyn.rs:1503]: 2 columns are enough

**Review by sheyanjie-taos — APPROVED** [review]
LGTM

File: taos-ws/src/client/mod.rs [hunk 2]
```diff
@@ -0,0 +1,623 @@
...
+                let host = get_host(&uri)?;
+                if cfg!(feature = "rustls") {
+                    #[cfg(feature = "rustls")]
+                    {
+                        let stream = connector::wrap_rustls(stream, host, self.certs.clone())?;
+                        builder.with_stream(uri, stream::SyncStream::Rustls(stream), check_fn)
+                    }
+                    #[cfg(not(feature = "rustls"))]
+                    {
+                        panic!("")
```
[the remark this answers is not in the document]
Comment (AdamEECS) [reply taos-ws/src/client/mod.rs]: done

**Review by zitsen — APPROVED** [review]
LGTM

**Comment by zitsen** [comment]
Remaining features will be implemented with another pr.

**Pull request merged**